docs/source/conf.py

```diff
@@ -64,7 +64,6 @@
 # -- Options for markdown extension
 scylladb_markdown_enable = True
 scylladb_markdown_recommonmark_versions = ['v1.9', 'v1.10']
-myst_heading_anchors = 6
```

Comment:
This line is added in the first commit and then removed in the second. Please squash the commits.